solr/test-framework/src/java/org/apache/solr/SolrJettyTestBase.java

```diff
@@ -160,8 +165,10 @@ public SolrClient createNewSolrClient() {
     }
   }
 
-  // Sets up the necessary config files for Jetty. At least some tests require that the solrconfig from the test
-  // file directory are used, but some also require that the solr.xml file be explicitly there as of SOLR-4817
+  // Sets up the necessary config files for Jetty. At least some tests require that the solrconfig
```

Comment:
comment style here doesn't flow well as Javadoc; maybe convert to Javavdoc, or reflow manually